frontend/src/components/PostForm.tsx

```diff
@@ -0,0 +1,29 @@
+import { Avatar, Button, Flex, HStack, VStack } from '@yamada-ui/react';
+import PostField from './PostField';
+import { useCallback, useState } from 'react';
+
+const PostForm = () => {
+  const [inputValue, setInputValue] = useState('');
+
+    const handleInputChange = useCallback((value: string) => {
+      setInputValue(value);
+    }, []);
+
+  return (
+    <div>
+      <VStack px={8}>
+        <HStack mt={12}>
+          <Avatar size="md" name="Hirotomo Yamada" />
+          <PostField onInputChange={handleInputChange} />
+        </HStack>
+        <Flex justify={'right'} my={8}>
```

Comment:
デザイン的には投稿フィールドとポストするボタン近いほうがTwitterに近いと思う

```suggestion
        <Flex justify={'right'} my={0}>
```

Reply:
あーモーダルの方のポストするボタンの下余白が消えるのか

こうしたらいいのかな
```tsx
<VStack padding={8}>
```

Reply:
確かに本家よりも余白が大きいかも！
修正します！

**スクリーンショット**
<img width="324" alt="image" src="https://github.com/givery-bootcamp/dena-2024-team4/assets/56498440/0aff4a99-ef2f-4a17-899a-91176b54ed7d">


**変更後のコード**
```    <VStack padding={8}>
      <HStack>
        <Avatar size="md" name="Hirotomo Yamada" />
        <PostField onInputChange={handleInputChange} />
      </HStack>
      <Flex justify={'right'}>
        <Button colorScheme="sky" rounded="24px" isDisabled={!inputValue}>
          ポストする
        </Button>
      </Flex>
    </VStack>
```